src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -189,6 +189,39 @@
     type: command
     short-summary: List policy definitions.
 """
+helps['policy setdefinition create'] = """
+            type: command
+            short-summary: Create a policy set definition.
+            parameters:
+                - name: --definitions
+                  type: string
+                  short-summary: Policy definitions in JSON format, or a path to a file containing JSON rules.
```

Comment:
This should be a json, since it takes a json array which can have:
1) policy definition id or
2) policy rule

With regards to help, the definitions can also be a uri that points to a valid json file, just like the rules in policy definition

Reply:
Since you are going for parity with `policy definition create` then disregard my comment. However, do make sure the help and behavior is consistent.